build.go

```diff
@@ -255,6 +257,26 @@ func (cmd *buildCommand) Run(args []string) (err error) {
 	eg.Go(func() error {
 		return sess.Run(ctx, sessDialer)
 	})
+
+	var cacheExports []*controlapi.CacheOptionsEntry;
+	cacheExports = append(cacheExports, &controlapi.CacheOptionsEntry{
+		Type:  "inline",
+	});
+
+	var cacheImports []*controlapi.CacheOptionsEntry;
+	for _, cacheTarget := range cmd.cacheFrom.GetAll() {
+		cacheImports = append(cacheImports, &controlapi.CacheOptionsEntry{
+			Type:  "registry",
+			Attrs: map[string]string{
+				"ref": cacheTarget,
+			},
+		});
+    }
```

Comment:
I do not understand why this is insufficient by itself and still requires the `cache-from` frontendAttrs to be passed through